src/core_ocean/analysis_members/Registry_mixed_layer_depths.xml

```diff
@@ -77,7 +77,16 @@
 			 description="mixed layer depth based on gradient of density"
 		/>
 	</var_struct>
-	<streams>
+<var_struct name="mixedLayerDepthAMScratch" time_levs="1">
+  <var name="pressureAdjustedForLandIceScratch"
+▸\▸\▸\ persistence="scratch"
```

Comment:
I'm still seeing weird non-tab characters here.  Could you see if you can fix those?

Reply:
@xylar I think I got the white space figured out and fixed.